Cargo.toml

```diff
@@ -22,7 +22,7 @@ include = [
 byteorder = "1.3.1"
 errno = "0.2.4"
 structopt = "0.2.11"
-wabt = "0.7.2"
+wabt = { git = "https://github.com/pepyakin/wabt-rs", rev = "60c196ce2b946e271e66da80f413d84049ba0a76" }
```

Comment:
Is there a publishability concern here if this is got based?

Reply:
Yes. I asked @MarkMcCaskey and he said it'd be fine to break wasmer-llvm-backend's pushability since it's never been pushed, but we shouldn't break the rest of wasmer right now.

Reply:
Yeah, we can deal this when we want to deal with publishing our LLVM work as a whole.  I figure that it won't be for quite a while, so it doesn't really matter, many of these changes will probably be upstreamed!

Good catch though, Brandon!